lib/rex/proto/smb/constants.rb

```diff
@@ -1398,6 +1398,17 @@ def self.make_nbs (template)
     [ 'Payload', 'ByteCount',  nil, true ]
   )
 
```

Comment:
@dmohanty-r7 / @sgonzalez-r7 this is something that needs to be added to Ruby_SMB no?